tests/test_for_support/test_for_basic.py

```diff
@@ -480,6 +480,44 @@ def dup_rxns_multiple_list_anns_no_match(base):
     return base
 
 
+@register_with(MODEL_REGISTRY)
+def rxns_no_exchange(base):
+    """Provide a model with no exchange reactions"""
+    met_a = cobra.Metabolite("a_c", compartment="c")
+    met_b = cobra.Metabolite("b_c", compartment="c")
+    rxn_1 = cobra.Reaction("rxn1")
+    rxn_1.add_metabolites({met_a: -1, met_b: 1})
+    return base
+
+
+@register_with(MODEL_REGISTRY)
+def rxns_with_two_substrates(base):
+    """Provide a model with two substrates that can be taken up"""
+    met_a = cobra.Metabolite("a_c", compartment="c")
+    met_b = cobra.Metabolite("b_c", compartment="c")
+    met_c = cobra.Metabolite("c_e", compartment="e")
+    met_d = cobra.Metabolite("d_e", compartment="e")
+    rxn_1 = cobra.Reaction("rxn1")
+    rxn_1.add_metabolites({met_a: -1, met_b: 1})
+    base.add_boundary(met_c, reaction_id="EX_c_e", lb=-1000, ub=1000)
+    base.add_boundary(met_d, reaction_id="EX_d_e", lb=-1000, ub=1000)
+    return base
+
+
+@register_with(MODEL_REGISTRY)
+def rxns_with_two_false_substrates(base):
+    """Provide a model with two substrates that can be taken up"""
+    met_a = cobra.Metabolite("a_c", compartment="c")
+    met_b = cobra.Metabolite("b_c", compartment="c")
+    met_c = cobra.Metabolite("c_e", compartment="c")
+    met_d = cobra.Metabolite("d_e", compartment="c")
+    rxn_1 = cobra.Reaction("rxn1")
+    rxn_1.add_metabolites({met_a: -1, met_b: 1})
+    base.add_boundary(met_c, reaction_id="EX_c_c", lb=-1, ub=0)
```

Comment:
On the `type` issue please see my comments above.

If you change this, `Model.medium` will identify the two exchanges correctly, and essentially this test wont show anything else than the `rxns_with_two_substrates`. Hence, I think it ought to just be removed.